WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -206,7 +208,8 @@ + (NSArray *)routes
 
 + (id<FBResponsePayload>)handleDoubleTapCoordinate:(FBRouteRequest *)request
 {
-  XCUICoordinate *doubleTapCoordinate = [self.class getGestureCoordinate:request];
+  CGPoint doubleTapPoint = CGPointMake((CGFloat)[request.arguments[@"x"] doubleValue], (CGFloat)[request.arguments[@"y"] doubleValue]);
+  XCUICoordinate *doubleTapCoordinate = [self.class gestureCoordinateWithCoordinate:doubleTapPoint application:request.session.application shouldApplyOrientationWorkaround:YES];
```

Comment:
Sorry, please ignore my recent comment. This is done, because in iOS 9.3 drag coordinates are also inverted. So the current implementation works properly if one provides absolute coordinates based on the actual screen width/height, but will not work if relative coordinates based on some element position are provided. I think the first case is more relevant for such method call. If one wanted to have a method that clicks some element by relative coordinates, then he would create some other method (something like _tap_). But this method only accepts absolute coordinates

@imurchie waht is you opinion on it?